go.mod

```diff
@@ -50,8 +50,6 @@ require (
 	github.com/imdario/mergo v0.3.16 // indirect
 	github.com/josharian/intern v1.0.0 // indirect
 	github.com/json-iterator/go v1.1.12 // indirect
-	github.com/kubeflow/common v0.4.6 // indirect
-	github.com/kubeflow/mpi-operator v0.3.1-0.20230228224311-5946ef415759 // indirect
```

Comment:
Taking these two out of here is huge. Awesome.

Reply:
Yea, I was surprised by how much was impacted!